src/cloud-api-adaptor/gcp/README.md

```diff
@@ -102,6 +111,11 @@ peer-pods-ctlplane-0 Ready    control-plane,master   6m8s    v1.25.3
 peer-pods-worker-0   Ready    worker                 2m47s   v1.25.3
 ```
 
+> [!NOTE]
+> When using GKE, ensure the worker nodes have the appropriate labels configured:
+> * node-role.kubernetes.io/worker=""
```

Comment:
Hi @beraldoleal !

The CoCo operator no longer requires the "node-role.kubernetes.io/worker" label

Reply:
Thanks. fixed